builtin/settingtypes.txt

```diff
@@ -508,6 +508,8 @@ hud_hotbar_max_width (Maximum hotbar width) float 1.0
 #    Enables caching of facedir rotated meshes.
 enable_mesh_cache (Mesh cache) bool false
 
+mesh_generation_interval (Mesh generation interval) int 15 0 50
```

Comment:
@Fixer-007 can you write a description for this? I’m not very good in documenting.